core/domain/user_services.py

```diff
@@ -689,7 +689,7 @@ def get_system_user():
     """Returns user object with system committer user id.
 
     Returns:
-        system_user: user object with system committer user id.
+        system_user. user object with system committer user id.
```

Comment:
system_user is not a type. Please fix. Also the description should be capitalized.